### PR TITLE
labels filter interface changed in checkmk

### PR DESCRIPTION
**Pull request by BenediktSeidl**

### CODE REVIEW

File: tests/unit/webapi.test.ts [hunk 1]
```diff
@@ -21,18 +22,31 @@ const rs: RequestSpec = {
...
   siteopt: { site: 'ut_site' },
   hostregex: { host_regex: 'ut_host_regex' },
   service: { service: 'Check_MK' },
   serviceregex: { service_regex: 'ut_service_regex' },
-  host_labels: { host_label: '[{"value":"cmk/site:cmk210d"}]' },
+  host_labels: {
+    host_labels_1_bool: 'and',
+    host_labels_1_vs_1_bool: 'and',
+    host_labels_1_vs_1_vs: 'cmk/site:cmk210d',
+    host_labels_1_vs_count: '2',
```
Comment (oKenneth) [inline tests/unit/webapi.test.ts:36]: `host_labels_1_vs_count: '1',` right?

**Review by oKenneth — COMMENTED** [review]
LGTM, tiny nit

File: tests/unit/webapi.test.ts (hunk 1 above)
Comment (BenediktSeidl) [reply tests/unit/webapi.test.ts]: nice! i used the output generated by the ui as a basis (which creates an additional element at the end), but seems i mixed something up.

Comment (oKenneth) [reply tests/unit/webapi.test.ts]: Yeah, the UI needs to always render an additional empty element. But in configs like this we don't need to add that empty field

**Review by oKenneth — APPROVED** [review]

**Review by TheSpacyCat — APPROVED** [review]

**Pull request merged**